packages/components/src/components/table/table/table-styles.ts

```diff
@@ -1,41 +1,28 @@
-import {
-  getMediaQueryMin,
-  spacingStaticLarge,
-  spacingStaticMedium,
-  spacingStaticSmall,
-  textSmallStyle,
-} from '@porsche-design-system/utilities-v2';
+import { textSmallStyle, spacingFluidMedium } from '@porsche-design-system/utilities-v2';
 import { getCss } from '../../../utils';
-import {
-  addImportantToEachRule,
-  getFocusJssStyle,
-  getThemedColors,
-  hostHiddenStyles,
-  pxToRemWithUnit,
-} from '../../../styles';
+import { addImportantToEachRule, hostHiddenStyles, doGetThemedColors } from '../../../styles';
+import type { Theme } from '../../../types';
 
-const { primaryColor } = getThemedColors('light');
+export const cssVariableTableRowHoverColor = '--p-internal-table-row-hover-color';
+export const cssVariableTableBorderColor = '--p-internal-table-border-color';
+export const cssVariableTableTheme = '--p-internal-table-theme';
+export const getComponentCss = (theme: Theme): string => {
+  const { primaryColor, backgroundSurfaceColor, contrastLowColor } = doGetThemedColors(theme);
 
-export const getComponentCss = (): string => {
   return getCss({
     '@global': {
       ':host': addImportantToEachRule({
         display: 'block',
         ...hostHiddenStyles,
       }),
+      '::slotted(*)': addImportantToEachRule({
+        [cssVariableTableRowHoverColor]: backgroundSurfaceColor,
+        [cssVariableTableBorderColor]: contrastLowColor,
+        [cssVariableTableTheme]: theme,
```

Comment:
We don't need `cssVariableTableTheme` css variable as soon as sort button is reverted. Instead we need something like `cssVariableSortButtonColor`

Reply:
yes, I'll change it

Reply:
Done